### DISCUSSION TITLE
Alias for audio output device names

### DISCUSSION THREAD

**Discussion by HaArD2000** [File-New-Project/EarTrumpet#773] [Q&A] [answered]
I have multiple monitors that are recognized as Audio devices and whenever I want to output sound to one of those monitors I have to go through several attempts to select the correct one because they all have the exact same name in Ear Trumpet.

I'd love to have the ability to apply a persistent custom alias name to each audio device that would allow me to instantly know which audio device was which.

![EarTrumpetPic](https://user-images.githubusercontent.com/32270210/109397552-32434980-7905-11eb-816e-d69110e2e03f.png)

**Comment by davuxcom** [comment, marked as answer]
Change the display name by going into `Playback Devices` and edit the properties for the device.

![image](https://user-images.githubusercontent.com/1456216/109403718-0ed3cb00-7914-11eb-8ead-5242189c7e01.png)

![image](https://user-images.githubusercontent.com/1456216/109403716-08ddea00-7914-11eb-94c4-0769e7bdc555.png)

![image](https://user-images.githubusercontent.com/1456216/109403761-54909380-7914-11eb-8294-9852a31a5832.png)

**Comment by HaArD2000** [comment]
Thank you!!!!  Thank you!!!!  Thank you!!!!  Thank you!!!!  Thank you!!!!  Thank you!!!!
